### ISSUE TITLE
Client Expirations

### ISSUE THREAD

**Issue by akoebbe** [happybottoms/coverd#85]
My initial plan was to make this very configurable, but for the sake of making functional progress I think we just need to make this simple.

Clients have several ways of "aging" out of getting products.

1. By age: 4 years old (status is flipped the 1st of the month after their birthday)
1. By distribution window: 3 years since first distribution (status is flipped the 1st of the month after their 3-year anniversary of their first distribution)
1. Pull-up max: Clients only get 6 pull-up distributions. After that pull-ups are no longer an option, and they would have to go back to diapers.

These expirations can be overridden by HB and customized to something else. See Drupal screenshot below...

<img width="806" alt="Screen Shot 2020-01-15 at 9 59 02 PM" src="https://user-images.githubusercontent.com/1044465/72492587-5caa7400-37e3-11ea-8bfe-b3fde8033d46.png">

I think we could make the different expirations configuable. Age-out, distribution max, and pullup max could be in the admin and could be changed app wide. But we can make that another ticket if that makes more sense.

**Comment by akoebbe** [comment]
Pausing on this one until #86 is complete

**Comment by akoebbe** [comment]
Scratch that. I forgot about the pull-up max limit.

**Comment by akoebbe** [comment]
OK. All calculation functions are in. Just need to wait on the workflow to trigger transitions.
